styles/Main.css

```diff
@@ -90,3 +150,45 @@ input:invalid {
 input:valid {
   background-color: none;
 }
+
+iframe{
+  border-top-right-radius: .5cm;
+  border-bottom-left-radius: .5cm;
+}
+
+footer{
+  font-size: 75%;
+  text-align: center;
+}
+
+a[href="#top"] {
```

Comment:
No need to add a class to the back-to-top link. You can use an attribute selector to select it specifically